File: js/padArray.js
// REMEMBER TO PSEUDOCODE
const pad = (array, minSize, value = null) => {
  if (array.length === minSize || minSize === 0) {
    return array;
  }
  for (let i = 0; i <= minSize - array.length; i++) {
    if (value === null) {
      array.push("None");
    } else {
      array.push(value);
    }
  }
  return array;
};

console.log(pad([1, 2, 3], 5));
console.log(pad([1, 2, 3], 5, "apple"));
console.log(pad([1, 2, 3], 3));

File: python/pad_array.py
#REMEMBER TO PSEUDOCODE
def pad(array, min_size, value = None):
    if len(array) == min_size or min_size == 0:
        return array
    for i in range(min_size - len(array)):
        if value == None:
            array.append("None")
        else:
            array.append(value)
    return array

# print(pad([1,2,3], 5))
# print(pad([1, 2, 3], 5, "apple"))
# print(pad([1, 2, 3], 3))